views/res_partner_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">openacademy.res.partner.view.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='active_lang_count']" position="after">
+                <field name="instructor" string="Instructor"/>
```

Comment:
string is not required here AFAIK, as it's already the same as the field's string.

Same below.